ci-templates/gitlab/image-builder.yml

```diff
@@ -51,58 +81,62 @@ frontend:
   rules:
     - if: '$FRONTEND == "1"'
       when: always
-  variables:
-    BASE_IMAGE: $DOCKER_REGISTRY/base
-    IMAGE: $DOCKER_REGISTRY/capella/collab/frontend
   script:
+    - *decrypt
+    - BASE_IMAGE=${DOCKER_REGISTRY}/base
+    - IMAGE=${DOCKER_REGISTRY}/capella/collab/frontend
+    - DOCKER_TAG=$(echo ${REVISION} | sed 's/[^a-zA-Z0-9.]/-/g')
```

Comment:
This can be reused with a yaml anchor as it used in all jobs

Reply:
Not like it is right now as the `IMAGE=...` is different for all of the jobs. Anyway, at least to me it looks like we can safely move this to the end of this block and move the general stuff to a `prepare` anchor.